components/vc/rear/src/HW/HW_gpio_componentSpecific.c

```diff
@@ -41,14 +216,183 @@ const HW_GPIO_S HW_GPIO_pinmux[HW_GPIO_COUNT] = {
         .mode = GPIO_MODE_AF_PP,
         .speed = GPIO_SPEED_FREQ_HIGH,
         .pull = GPIO_NOPULL,
-        .resetState = HW_GPIO_PINSET,
+        .resetState = HW_GPIO_PINRESET,

```

Comment:
Tbh im not sure what the reset tx state of can is. I have tried looking into it in the passed it might be a set state. Wed really have to check like stm cube ide or find it in the rm